packages/sit-onyx/src/components/OnyxCheckbox/OnyxCheckbox.vue

```diff
@@ -37,6 +44,33 @@ const isChecked = computed({
 </template>
 
 <style lang="scss">
+@mixin define-hover-border($state, $color) {
+  .onyx-checkbox__input {
+    &#{$state} {
+      border-color: var(--onyx-color-base-#{$color}-300);
+    }
+  }
+}
```

Comment:
I guess you could make the selector a oneliner?

```suggestion
  .onyx-checkbox__input #{$state} {
    border-color: var(--onyx-color-base-#{$color}-300);
  }
}
```